ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -512,15 +519,15 @@ export const DataAccessRequirements = fp.flow(withProfileErrorModal)((spinnerPro
         <div style={styles.selfBypassText}>[Test environment] Self-service bypass is enabled</div>
         <Button
             style={{marginLeft: '0.5rem'}}
-            onClick={() => selfBypass(spinnerProps, reload)}>Bypass all</Button>
+            onClick={async() => await selfBypass(spinnerProps, reload)}>Bypass all</Button>
```

Comment:
opt: this async/await I don't think does anything, since there's nothing else to do here after `selfBypass`, it's basically a fire/forget request in both cases.